src/ResourceManager/Sql/Commands.Sql/Database/Cmdlet/NewAzureSqlDatabase.cs

```diff
@@ -101,6 +100,11 @@ public class NewAzureSqlDatabase : AzureSqlDatabaseCmdletBase
         [Alias("Tag")]
         public Hashtable Tags { get; set; }
 
+        [Parameter(Mandatory = false,
+            HelpMessage = "The name of the sample schema to apply when creating this database.")]
+        [ValidateNotNullOrEmpty]
+        public string SampleName { get; set; }
```

Comment:
@jaredmoo would you mind providing an example of when a user would use this parameter? Also, how does the user discover which sample schemas they may provide to this cmdlet? I just want to make sure I understand the scenario trying to be covered by this addition and if `ValidateSet` should be used (if there is a predetermined set of schemas that can be used)